projects/Mallard/src/components/article/html/header.ts

```diff
@@ -239,22 +239,8 @@ export const headerStyles = ({
     }
 `
 
-const Image = ({
-    image,
-    publishedId,
-    className,
-}: {
-    publishedId: Issue['publishedId']
-    image: ImageT
-    className?: string
-}) => {
-    const backend = defaultSettings.apiUrl
-    const path = `${backend}${mediaPath(
-        publishedId,
-        imageForScreenSize(),
-        image.source,
-        image.path,
-    )}`
+const Image = ({ image, className }: { image: ImageT; className?: string }) => {
+    const path = useImagePath(image)
```

Comment:
Just as a head up: just revisited this line and it's pretty odd, not sure how're allowed to call a hook here without an error?